### ISSUE TITLE
Add option not to show '+' links by default

### ISSUE THREAD

**Issue by 3v1n0** [benwinding/newsit#7]
When a page is not found a link with a '+' item is added to each page, while would be nicer not to show anything in such case.

**Comment by benwinding** [comment]
That's a great idea @3v1n0, thanks for the suggestion, I'll build it into the next release :+1:

**Comment by benwinding** [comment]
Deployed immediately to firefox extensions store, but can take days/weeks to be deployed on Chrome store

**Comment by 3v1n0** [comment]
Thanks works great!

The only thing is that when you go to "NewsIt" pane and click "Check current page", maybe the + should be added?

I.e.: show it only on explicit request.

**Comment by benwinding** [comment]
I agree, that's a good idea too. Will leave this open till it's complete
